tracer/src/Datadog.Trace.SourceGenerators/TelemetryMetric/Sources.Attributes.cs

```diff
@@ -27,13 +27,38 @@ internal sealed class TelemetryMetricTypeAttribute : System.Attribute
             /// Initializes a new instance of the <see cref="TelemetryMetricTypeAttribute"/> class.
             /// </summary>
             /// <param name="metricType">The type of the metric, e.g. Count, Gauge</param>
-            public TelemetryMetricTypeAttribute(string metricType) =>
-                this.MetricType = metricType;
+            public TelemetryMetricTypeAttribute(string metricType)
+                : this(metricType, false, true)
+            {
+            }
+
+            /// <summary>
+            /// Initializes a new instance of the <see cref="TelemetryMetricTypeAttribute"/> class.
+            /// </summary>
+            /// <param name="metricType">The type of the metric, e.g. Count, Gauge</param>
+            /// <param name="isCiVisibilityMetric">Should the metrics be emitted by the CI Visibility collector?</param>
+            /// <param name="isApmMetric">Should the metrics be emitted by the standard (APM) collector?</param>
+            public TelemetryMetricTypeAttribute(string metricType, bool isCiVisibilityMetric, bool isApmMetric)
```

Comment:
I guess it's ok if we have only those two, but if we add new ones we should probably switch to a flags enum